vuepress-docs/docs/.vuepress/public/QForm/[QFormItem]labelSize.html

```diff
@@ -0,0 +1,70 @@
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <!-- import CSS -->
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@qvant/qui-max/dist/style.css"
+    />
+  </head>
+
+  <body>
+    <div id="app">
+      <div class="block">
+        <q-row>
+          <q-col cols="6">
+            <q-form
+              ref="form"
+              :model="formModel"
+            >
+              <q-form-item
+                label="Name"
```

Comment:
```suggestion
                label="Small label"
```

и добавь рядом пример с regular